Assets/Resources/Prefabs/Popups/DamagePopup.prefab

```diff
@@ -78,7 +78,7 @@ MeshRenderer:
   m_AutoUVMaxAngle: 89
   m_LightmapParameters: {fileID: 0}
   m_SortingLayerID: -1243974647
-  m_SortingLayer: 2
+  m_SortingLayer: 3
```

Comment:
i dont actually recall doing this...it shouldn't matter it just means the popup is extra in front now, but that's interesting